doc/modules/mod_mam.md

```diff
@@ -6,117 +6,83 @@ Configure MAM with different storage backends:
 * ODBC (RDBMS, like MySQL, PostgreSQL, MS SQL Server)
 * Riak KV (NOSQL)
 
-### Configure MAM with ODBC backend
 
-#### Options
-* `mod_mam_odbc_prefs`, `mod_mam_mnesia_prefs`:
-Consider the process as a kind of recipe. For each step you can enable none ("optional"), one ("single") or more ("multi") modules, according to instructions. Provided there are any, please use the options described in a specific step. All config parameters are boolean, so you can enable them by adding an atom to the configuration list, e.g. `{mod_mam_odbc_arch, [pm, no_writer]}`.
+`mod_mam_meta` is a meta-module that ensures all relevant `mod_mam_*` modules are loaded and properly configured.
 
-##### Step 1 (multi)
-* `*mod_mam` + `mod_mam_odbc_arch`: Enables support for one-to-one messages archive.
-* `mod_mam_muc` + `mod_mam_muc_odbc_arch`: Enables support for group chat messages archive.
+### Options
 
-If you haven't chosen any of the above, skip the next part.
+* **backend** (atom, default: `odbc`) - Database backend to use. `odbc`, `riak` and `cassandra` are supported.
+* **add_archived_element** (atom, default: `false`) - Add `<archived/>` element from MAM v0.2.
+* **is_complete_message** (module, default: `mod_mam_utils`) - Name of a module implementing [`is_complete_message/3` callback](#is_complete_message) that determines if the message should be archived.
```

Comment:
Maybe let's refactor this to `is_archivable_message`, because `complete` doesn't really makes sense IMO.